pypredict/trades.py

```diff
@@ -60,7 +55,7 @@ async def recv_and_publish(self, uri):
                     while True:
                         message = await websocket.recv()
                         for event in self.message_to_events(json.loads(message)):
-                            await self.ensign.publish(self.topic, event, ack_callback=handle_ack, nack_callback=handle_nack)
+                            await self.ensign.publish(self.topic, event, ensure_exists=True, on_ack=handle_ack, on_nack=handle_nack)
```

Comment:
`ensure_exists` is necessary now even if the topic exists as you probably figured out but I have a PR open that should fix that.

Reply:
The other workaround is to use the topic ID.

Reply:
I updated the code to use the topic ID